### PR TITLE
semgrep 0.20.0

### PR DESCRIPTION
**Pull request by minusworld**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/semgrep.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class Semgrep < Formula
   desc "Easily detect and prevent bugs and anti-patterns in your codebase"
   homepage "https://semgrep.dev"
   url "https://github.com/returntocorp/semgrep.git",
-    tag:      "v0.19.1",
-    revision: "e9e42cf838790741bda870735b942cc736b80fd2"
+    tag:      "v0.20.0",
+    revision: "1e89dcc5d1e88d447397b663d5de7ce2a38976c0"
   license "LGPL-2.1"
```
Comment (brendongo) [inline Formula/semgrep.rb:9]:
```suggestion
  license "LGPL-2.1-only"
```
to get rid of deprecated license label

**Comment by chenrui333** [comment]
@minusworld, thanks for the PR, can you rebase and do the two commits with commit messages? Thanks!

- `semgrep 0.20.0`
- `semgrep: update license`

**Review by chenrui333 — APPROVED** [review]
💯

**Comment by chenrui333** [comment]
Thanks @minusworld @brendongo !!

**Pull request closed without merging**